common/djangoapps/microsite_configuration/__init__.py

```diff
@@ -1 +1,19 @@
+from django.conf import settings as base_settings
+
+from microsite_configuration import microsite
 from .templatetags.microsite import page_title_breadcrumbs
+
+
+class MicrositeAwareSettings():
+    """
+    This class is a handy utility to make a call to the settings
+    completely microsite aware by replacing the:
+    from django.conf import settings
+    with:
+    from microsite_configuration import settings
+    """
+
+    def __getattr__(self, name):
+        return microsite.get_value(name, base_settings.__getattr__(name))
```

Comment:
One thing all users of microsites will have to do is to - for the short term - provide duplicate definitions inside of MICROSITE_CONFIGURATION for "platform_name" == "PLATFORM_NAME". I don't think it should be within the scope of these changes to touch all the points where we have to change the capitalization.

We'll have to change our configuration on our sites at edx.org, so this PR will have to be coordinated with some DevOps changes.

Either that or have some ugly logic somewhere (say in get_value()) which treats "platform_name" as "PLATFORM_NAME".

Not sure which is better/worse.